dogs/dogs.js

```diff
@@ -8,12 +8,9 @@ window.addEventListener('load', async() => {
     const id = new URLSearchParams(window.location.search).get('id');
     
     const dog = await getDog(id);
-    console.log(dog);
     
     const dogEl = renderDogDetail(dog);
 
     dogContainerEl.append(dogEl);
-
-
 });
 
```

Comment:
In your index.html in the dogs folder, you are missing a dot in your path to the home page which is causing your home button link to fail.  Otherwise nice job rebuilding the details folder.